oozie/src/main/java/org/apache/falcon/oozie/OozieBundleBuilder.java

```diff
@@ -106,11 +108,24 @@ public Properties build(Cluster cluster, Path buildPath) throws FalconException
         properties.setProperty(AbstractWorkflowEngine.NAME_NODE, ClusterHelper.getStorageUrl(cluster));
 
         //Add libpath
-        String libPath = getLibPath(buildPath);
-        if (libPath != null) {
-            properties.put(OozieClient.LIBPATH, getStoragePath(libPath));
+        String path = getLibPath(buildPath);
```

Comment:
I don't understand your comment completely. Are you saying getStoragePath should return comma separated lib paths? 

getStoragePath is called from multiple places and for other cases input is not comma spearated list and hence becomes unnecessary processing. Hence I moved the logic only where WF libPaths are processed. You can verify all the callers of getStoragePath.